AGGREGATE.md

```diff
@@ -187,8 +189,22 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
-**TODO**: more formally specify `payload`.
+For the MPC protocol, we propose the contents of this base64-encoded encrypted payload will be a [CBOR](https://cbor.io) map with the following structure:
+
+```
+// CBOR
+{
+  "version": "<api version>",
+  "operation": "hierarchical-histogram",  // Allows for the service to support other operations in the future
+  "privacy_budget_key": "<field for server to do privacy budgeting>",
```

Comment:
Should we keep a TODO to more formally specify privacy_budget_key?

Reply:
Done